ProcessMaker/Events/PackageEvent.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace ProcessMaker\Events;
+
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Support\Collection;
+
+class PackageEvent
+{
+    use Dispatchable, InteractsWithSockets, SerializesModels;
+
+    public $packages;
+
+    /**
+     * Create a new event instance.
+     *
+     * @param Collection $packages
+     */
+    public function __construct(Collection $packages)
+    {
+        $this->packages = $packages;
+    }
+
+    /**
+     * Get the channels the event should broadcast on.
+     *
+     * @return \Illuminate\Broadcasting\Channel|array
+     */
+    public function broadcastOn()
+    {
+        return new PrivateChannel('channel-name');
```

Comment:
The broadcast channel name looks undefined (or using the default value 'channel-name') This should returns the channel that the event should broadcast on. Examples:

- `ProcessMaker.Models.User.123`  ==>   `channel for user 123 only`
- `notifications` ==> `A channel named notifications`

Could be useful to define a proper channel name for this event, and define how are going to listen it.